common/check_rules_with_0_completion_volume

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2013
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Martin Barisits, <martin.barisits@cern.ch>, 2014
+# - Eric Vaandering, <ewv@fnal.gov>, 2019
+
+"""
+Probe to check the backlog of stuck rules which have not moved per rse.
+"""
+
+import sys
+import traceback
+
+from rucio.core import monitor
+from rucio.db.sqla.session import BASE, get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if BASE.metadata.schema:
+    schema = BASE.metadata.schema + '.'
+else:
+    schema = ''
+
+if __name__ == "__main__":
+    try:
+        session = get_session()
+        sql = 'SELECT rse_expression,sum(bytes) as sum_bytes FROM {schema}DIDS join {schema}RULES on DIDS.name = RULES.name where (state!=\'O\' and locks_ok_cnt=0) group by rse_expression'.format(
```

Comment:
Just be aware that "rse expression" is not going to just be RSE. This could blow up as people use a number of other kinds of expressions